index.html

```diff
@@ -9997,6 +9997,12 @@ <h5>Presentational Role Inheritance</h5>
           <li>The user interface makes the meaning and purpose of each state apparent.</li>
           <li>The user interface provides a separate method for controlling each state.</li>
         </ul>
+		<p>
+			User agents MUST calculate an implicit value for <sref>aria-level</sref> from the document structure if the author does not explicitly specify a value.
+		</p>
+		<p>
+			Authors MAY use nested <rref>group</rref> elements to implicitly indicate level hierarchy. Otherwise, authors MUST ensure all <rref>treeitem</rref> elements in the <rref>tree</rref> have explicit values for <sref>aria-level</sref>, <sref>aria-posinset</sref>, and <sref>aria-setsize</sref>.
+		</p>
```

Comment:
This is really a nit, but it seems to me these two paragraphs (which are related to the structure of the dom/accessibility tree) should be below the other line that is edited, because we also mention `group` and structural stuff up there.

Otherwise it's "structure stuff" then "selected stuff" then "structural stuff".

Reply:
That makes sense, I think I was initially hesitant to put it towards the beginning b/c it seemed like it was addressing a relatively unusual use case. Though I think your point about grouping (heh) the relative structure info holds more weight.